### PR TITLE
Fix: https://github.com/Azure/autorest/issues/923

### PR DESCRIPTION
**Pull request by begoldsm**
This should fix issue: https://github.com/Azure/autorest/issues/923

### CODE REVIEW

File: Samples/azure-storage/Azure.Python/setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # prerequisite: setuptools
 # http://pypi.python.org/pypi/setuptools
 
-REQUIRES = ["msrest>=0.1.0", "msrestazure>=0.1.0"]
```
Comment (begoldsm) [inline Samples/azure-storage/Azure.Python/setup.py]: It appears as though whoever checked in python changes didn't re-run regenerate:expected. I am modifying these files as a result of running that after a sync and build.

Comment (amarzavery) [inline Samples/azure-storage/Azure.Python/setup.py]: @lmazuel - Can you take a look at this ?

Comment (amarzavery) [inline Samples/azure-storage/Azure.Python/setup.py]: @annatisch - Can you take a look at this?

Comment (annatisch) [inline Samples/azure-storage/Azure.Python/setup.py]: Yeah I think you're right - I probably just ran regenerate:expected:python/azurepython.
I will take care in future to regenerate the samples as well. Thanks!

**Pull request merged**